windows/test-app.mjs

```diff
@@ -0,0 +1,228 @@
+//@ts-check
```

Comment:
Can you run `yarn prettier --write windows/test-app.mjs` so we can get this file formatted?

Reply:
Also, why the `.mjs` extension? Is that something new? Can we use `.js` instead?

Reply:
Using `.js` gives warning ```
To load an ES module, set "type": "module" in the package.json or use the
.mjs extension.
```

Reply:
I think if you replace `import` with `require`, it should go away.